src/main/java/frc/robot/subsystems/ball/feeder/UltrasonicPoller.java

```diff
@@ -26,9 +28,8 @@
         this.ultrasonicSensor = sensor;
         this.time = time;
         this.distance = distance;
-        this.noBalls = false;
-        this.startTime = 0;
 
+        this.firstRun = false; // Not sure if this should be static
```

Comment:
It should not.

Reply:
Should it not be initialized to `true`, and then set by the first run when it happens? ie

```
boolean firstRun = true;

task() {
  if (firstRun) {
    this.firstRun = false;
    // do first-run-only things
  }
```

Reply:
Yep, that's a mistake